src/DnsHelperUI/MainForm.Designer.cs

```diff
@@ -236,13 +223,35 @@ private void InitializeComponent()
             this.btnChooseDns.UseVisualStyleBackColor = true;
             this.btnChooseDns.Click += new System.EventHandler(this.btnChooseDns_Click);
             // 
+            // cmbNics
+            // 
+            this.cmbNics.DropDownStyle = System.Windows.Forms.ComboBoxStyle.DropDownList;
```

Comment:
Could you please adjust the style of this box to be flat and the font size to be slightly bigger, e.g. 
```
this.cmbNics.FlatStyle = System.Windows.Forms.FlatStyle.Flat;
this.cmbNics.Font = new System.Drawing.Font("Microsoft Sans Serif", 10F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
this.cmbNics.Location = new System.Drawing.Point(12, 10);
this.cmbNics.Size = new System.Drawing.Size(326, 24);
```